airflow/providers/microsoft/azure/operators/container_instances.py

```diff
@@ -90,7 +89,6 @@ class AzureContainerInstancesOperator(BaseOperator):
     :param restart_policy: Restart policy for all containers within the container group.
         Possible values include: 'Always', 'OnFailure', 'Never'
     :param ip_address: The IP address type of the container group.
-    :param network_profile: The network profile information for a container group.
```

Comment:
This is breaking change.
Please set major version on `provider.yaml` we will also need entry in changelog.rst (top of the file) with the things that were broken and how to mitigate them.

Reply:
yes, still try a few things, will update the provider.yaml and changelog.rst  if needed

Reply:
yeah, updated provider.yaml and changelog